iosApp/Survey/Sources/Presentation/Modules/SurveySelection/AccountView/AccountView+DataSource.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  AccountView+DataSource.swift
+//  Survey
+//
+//  Created by Bliss on 22/12/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Combine
+import KMPNativeCoroutinesCombine
+import Shared
+
+extension AccountView {
+
+    final class DataSource: ObservableObject {
+
+        private let viewModel: AccountViewModel
+        private let coordinator: AccountCoordinator
+
+        @Published private(set) var viewState = AccountViewState()
+        @Published var showingLoading = false
```

Comment:
@blyscuit I think `isShowingLoading` or `showLoading` is better. What do you think? 🤔

Reply:
@doannimble Updated